frontend/javascripts/oxalis/view/left-border-tabs/controls_and_rendering_settings_tab.js

```diff
@@ -279,31 +331,29 @@ class ControlsAndRenderingSettingsTab extends PureComponent<ControlsAndRendering
           />
           <SwitchSetting
             label={
-              <React.Fragment>
-                {settingsLabels.fourBit}{" "}
-                <Tooltip title="Decrease size of transferred data by half using lossy compression. Recommended for poor and/or capped Internet connections.">
-                  <InfoCircleOutlined />
-                </Tooltip>
-              </React.Fragment>
+              <Tooltip title="Decrease size of transferred data by half, using lossy compression. Recommended for poor and/or capped internet connections.">
+                {settingsLabels.fourBit}
+              </Tooltip>
             }
             value={this.props.datasetConfiguration.fourBit}
             onChange={this.onChangeDataset.fourBit}
           />
           {Constants.MODES_ARBITRARY.includes(this.props.viewMode) ? null : (
             <SwitchSetting
-              label={settingsLabels.interpolation}
+              label={
+                <Tooltip title="Smooth the rendered data by interpolating color values, depending on zoom step">
```

Comment:
```suggestion
                <Tooltip title="Smooth the rendered data by interpolating color values.">
```

not sure if the zoom-step-info is really helpful :thinking: